internal/runtime/init/init_test.go

```diff
@@ -56,6 +56,19 @@ p = true { 1 = 2 }`
 			expectedMods: []string{mod1},
 			asBundle:     false,
 		},
+		{
+			note: "load duplicate files",
+			fs: map[string]string{
```

Comment:
Can we somehow replicate the issue here? It wasn't so much about stating the same for twice, but loading the current directory (`.`) and a file in it.